### PR TITLE
ROC-4065 Full admissions

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4065

### Change description
Update defendant's claim status views according to the prototypes

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by andrewwa-kainos** [comment]
Note: CCJ PDF downloads are still unavailable

File: src/main/features/dashboard/views/macro/caseHistory.njk [hunk 1]
```diff
@@ -86,8 +88,9 @@ ccjRequestedForDefendantClaimDetails %}
 {% endmacro %}
 
 {% macro caseHistoryForDefendantClaimDetails(claim) %}
-  {% for state in claim.stateHistory %}
+  {% if claim.stateHistory and claim.stateHistory[0] %}
```
[the remark this answers is not in the document]
Comment (timja) [reply src/main/features/dashboard/views/macro/caseHistory.njk]: also shouldn't a case history be a loop? or is it too complex?

File: src/main/features/dashboard/views/macro/claimStatus/ccjRequested.njk [hunk 2]
```diff
@@ -17,26 +17,29 @@
...
+      t(' if you need their payment details. Make sure you get receipts for any payments.')
+    }}
   </p>
+
+  <p>{{ t('The judgment sets out the terms of how you must pay.') }}</p>
+
+  <p>{{ t('View the judgment') }} {{ t(' (PDF)') }}</p>
+
+  <p>{{ t('If you repay the debt within 28 days you can <a href="{{ link }}">apply for a certificate of cancellation</a> to cancel the CCJ. This application costs £15.',
+      { link: 'https://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?court_forms_id=1157' }) | safe }}</p>
```
Comment (timja) [inline src/main/features/dashboard/views/macro/claimStatus/ccjRequested.njk]: should this say (external link) and have a target of `_blank`?

**Comment by robertparkinson** [comment]
closing as now part of #945

**Pull request closed without merging**